.github/workflows/ci.yml

```diff
@@ -37,23 +46,26 @@ jobs:
       run: |
         DIR="checkMPIdebug_shared" && mkdir -p "$DIR" && cd "$DIR"
         ../configure --enable-mpi --enable-debug --disable-shared \
-            CFLAGS="-O0 -g -Wall -Wextra -Wno-unused-parameter"
+            CFLAGS="-O0 -g -Wall -Wextra -Wno-unused-parameter" \
+            CPPFLAGS="-DSC_HACK_MPI_SLEEP=$IO_SLEEP_TIME"
         make -j V=0
         make -j check V=0
 
     - name: Make check with MPI, without debug
       shell: bash
       run: |
         DIR="checkMPI" && mkdir -p "$DIR" && cd "$DIR"
-        ../configure --enable-mpi CFLAGS="-O2"
+        ../configure --enable-mpi CFLAGS="-O2" \
+        CPPFLAGS="-DSC_HACK_MPI_SLEEP=$IO_SLEEP_TIME"
```

Comment:
Not to be too picky, but can you please indent this line by 4?

Reply:
Done